src/domain/ledger/index.ts

```diff
@@ -25,6 +26,11 @@ export const LedgerTransactionType = {
   ToHotWallet: "to_hot_wallet",
 } as const
 
+export const ExtendedLedgerTransactionType = {
```

Comment:
Why was this added as an additional type.

Reply:
It was to help us distinguish ln-intraledger from normal intraledger transactions since we currently lump both under `on_us` in the db whereas onchain-intraledger has its own `onchain_on_us` type.

It got used in the `tx-history` switch when categorizing different transactions into their respective `initiationVia` and `settlementVia` types.